### ISSUE TITLE
Is there a difference between those packetloss functions?

### ISSUE THREAD

**Issue by khalidahmedshalabi** [KHKKhalid/SAMPBulletproof#21]
Same accuracy?

**[1]** 

``` c++
forward Float:GetPlayerPacketLoss(playerid);
public Float:GetPlayerPacketLoss(playerid) {

    new stats[401], stringstats[70];
    GetPlayerNetworkStats(playerid, stats, sizeof(stats));
    new len = strfind(stats, "Packetloss: ");
    new Float:packetloss = 0.0;
    if(len != -1) {
        strmid(stringstats, stats, len, strlen(stats));
        new len2 = strfind(stringstats, "%");
        if(len != -1) {
            strdel(stats, 0, strlen(stats));
            strmid(stats, stringstats, len2-3, len2);
            packetloss = floatstr(stats);
        }
    }
    return packetloss;
}
```

**[2]** http://wiki.sa-mp.com/wiki/NetStats_PacketLossPercent

**Comment by Whitetigerswt** [comment]
Yes they'll output the same values.
